src/description.cpp

```diff
@@ -319,9 +555,33 @@ string Description::roleToString(Role role) {
 	}
 }
 
-} // namespace rtc
+std::_Rb_tree_iterator<std::pair<const int, Description::Media>> Description::getMedia(int mLine) {
+    return this->mMedia.find(mLine);
+}
+
+rtc::Description::Media& Description::addAudioMedia() {
+    rtc::Description::Media media("audio 9 UDP/TLS/RTP/SAVPF");
+    media.mid = "audio";
+
+    media.attributes.emplace_back("rtcp-mux");
+
+    this->mMedia.insert(std::pair<int, Media>(this->mMedia.size(), media));
+    return this->mMedia.at(mMedia.size()-1);
+}
+
+Description::Media& Description::addVideoMedia(Description::Direction direction) {
+    rtc::Description::Media media("video 9 UDP/TLS/RTP/SAVPF");
+    media.mid = "video";
+    media.attributes.emplace_back("rtcp-mux");
+
+    media.setDirection(direction);
+
+    this->mMedia.insert(std::pair<int, Media>(this->mMedia.size(), media));
+    return this->mMedia.at(mMedia.size()-1);
+}
 
 std::ostream &operator<<(std::ostream &out, const rtc::Description &description) {
 	return out << std::string(description);
 }
 
+}
```

Comment:
Moving `operator<<` inside `rtc` namespace shouldn't compile as it is declared outside in the header.